src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -42,6 +42,7 @@
 #include "utils/gen_hash.h"
 #include "utils/helpers.h"
 
+#define INVALID_IMG_WARNING_INFO "Requested image(s) not found."
```

Comment:
would prefer `const std::string`

Reply:
@jacobkeeler agree. But in this case `const char*` will be better as `SendResponse` requires char* as a parameter. See https://github.com/smartdevicelink/sdl_core/pull/2276/commits/51e40bf42f03ed368ac51123b77444313a2fe84d